test/test_config.rb

```diff
@@ -16,6 +17,7 @@ def test_default_max_threads
   end
 
   def test_app_from_rackup
+    skip_if :rack3
     conf = Puma::Configuration.new do |c|
       c.rackup "test/rackup/hello-bind.ru"
```

Comment:
Can we use the rackup gem instead of skipping the test? https://github.com/rack/rack/blob/main/UPGRADE-GUIDE.md (On phone atm so haven't checked myself)

Reply:
Loading a gem that has webrick as a dependency seems odd.  1,249 downloads since 03-Aug-22.

Reply:
Oh right, yeah never mind...